system/settings/rbp.xml

```diff
@@ -95,6 +95,7 @@
       </group>
       <group id="3">
         <setting id="audiooutput.ac3transcode" help="37024">
+          <default>false</default>
         </setting>
```

Comment:
Definitely want ac3 transcode disabled on Pi1.
I'm happy for Pi2/Pi3 to match whatever the default is chosen to be.